compose/ui/ui-test/build.gradle

```diff
@@ -40,33 +40,33 @@ android {
 }
 
 dependencies {
-    kotlinPlugin(project(":compose:compiler:compiler"))
+    kotlinPlugin(projectOrArtifact(":compose:compiler:compiler"))
 
     if (!AndroidXComposePlugin.isMultiplatformEnabled(project)) {
-        api(project(":compose:runtime:runtime"))
-        api(project(":compose:ui:ui"))
-        api(project(":compose:ui:ui-graphics"))
-        api(project(":compose:ui:ui-text"))
-        api(project(":compose:ui:ui-unit"))
+        api(projectOrArtifact(":compose:runtime:runtime"))
+        api(projectOrArtifact(":compose:ui:ui"))
```

Comment:
All of these are in this project and should stay as project("..

Reply:
Done